src/coreclr/jit/compiler.h

```diff
@@ -513,6 +513,10 @@ class LclVarDsc
     {
         return lvTracked && lvType != TYP_STRUCT;
     }
+#ifdef DEBUG
+    unsigned char lvTrackedWithoutIndex : 1; // Tracked but has no lvVarIndex (i.e. only valid GTF_VAR_DEATH flags, used
+                                             // by physical promotion)
+#endif
```

Comment:
This is just used to print "(last use)" in the dump outside the new locals introduced by physical promotion.